src/app/space/create/deployments/models/scaled-memory-stat.ts

```diff
@@ -14,6 +14,7 @@ export class ScaledMemoryStat implements MemoryStat {
   constructor(
     public readonly used: number,
     public readonly quota: number,
+    // FIXME: Currently timestamp unused?
```

Comment:
I don't think this is unused. `DeploymentsService#getDeploymentMemoryStat` returns `ScaledMemoryStat` objects, and there is at least one subscriber to this that uses the timestamp field in the `DeploymentDetailsComponent`. This component only expects the type to be the MemoryStat interface however, not the specific ScaledMemoryStat class, so this might make the timestamp field appear unreferenced at a glance.